packages/scrollregion/demo/stories/ScrollRegionStory.tsx

```diff
@@ -72,16 +72,12 @@ interface IArgs extends IScrollRegionContainerProps<HTMLDivElement> {
 }
 
 export const ScrollRegionStory: StoryFn<IArgs> = ({ as, ...props }) => {
-  const ScrollRegion = () => {
-    switch (as) {
-      case 'container':
-        return <Container {...props} />;
+  switch (as) {
+    case 'container':
+      return <Container {...props} />;
 
-      case 'hook':
-      default:
-        return <Hook {...props} />;
-    }
-  };
-
-  return <ScrollRegion />;
+    case 'hook':
+    default:
+      return <Hook {...props} />;
+  }
```

Comment:
[nit] Not sure a `switch` statement is needed for such case. I'll leave it up to you if you wanna refactor with a couple `if` statements.

Reply:
I ended up stripping the inline component and left the switch, but I'm not opposed to replacing with an `if` condition for `as === 'container'` with a default render using the `Hook` component.